azkaban-exec-server/src/main/java/azkaban/container/FlowContainer.java

```diff
@@ -113,7 +109,6 @@
   private static final String DEFAULT_LOG_CHUNK_SIZE = "5MB";
   private static final int DEFAULT_LOG_NUM_FILES = 4;
   private static final int EXEC_ID_INDEX = 0;
-  private static final int PROJECT_LOCATION_INDEX = 1;
```

Comment:
Can you please verify that other indexes are correct after removing this one?

Reply:
There is just one which is exec id and that too will be removed in subsequent PRs when it is picked from environment.